model.py

```diff
@@ -10453,15 +10455,15 @@ def default_loan_period(self):
         """
         return (
             self.external_integration.setting(
-                self.DEFAULT_LOAN_PERIOD_KEY).int_value
+                self.EBOOK_LOAN_DURATION_KEY).int_value
```

Comment:
This doesn't look like it's per-library?

Reply:
It's the default for the collection but libraries can change it?

Reply:
Aaaargh.

I don't know how to solve this without ripping up the whole estimate system -- a system that we know would remain broken in many ways even if this were fixed.